newtest/unittest1.ts

```diff
@@ -0,0 +1,69 @@
+import fetchMock, { MockResponse } from "fetch-mock";
```

Comment:
Since, "end" and "data" events are used to capture the outputs. We should use the actual fetch of node-fetch and wrap it using wrappedNodeFetch function.